### ISSUE TITLE
Cannot select and copy text in votes

### ISSUE THREAD

**Issue by lilydjwg** [telegramdesktop/tdesktop#5979]
### Steps to reproduce
1. start a vote
2. try to copy text of the options

### Expected behaviour
I should be able to copy them

### Actual behaviour
I couldn't find a way to copy them

### Configuration
**Operating system:**
Arch Linux
**Version of Telegram Desktop:**
1.6.7-1
**Used theme**:
Blue

**Comment by omoridi** [comment]
I think add edit option for vote. maybe some words was wrong

**Comment by lilydjwg** [comment]
Yes editing is helpful, but copying is still needed, e.g. other people may want to copy (part of) an option for discussing.

**Comment by lilydjwg** [comment]
I'm still experiencing this issue and no easy workarounds (I've got two workaround: one is OCR which is inaccurate the other is to write a program to use the API, bypassing this client).

**Comment by lilydjwg** [comment]
> We therefore assume that the user has lost interest or resolved the problem on their own.

No, I'm still experiencing this issue from time to time. The workaround is to ask the vote poster to post the text again in a normal message (or type it by myself). Both needs human effort every time that could be reduced.
